pkg/serverconfig/types.go

```diff
@@ -58,3 +59,11 @@ type Customization struct {
 type Providers struct {
 	StatuspageID string `yaml:"statuspageID,omitempty"`
 }
+
+type HelmChartRepo struct {
+	Url string `yaml:"url,omitempty"`
+}
+
+type Helm struct {
+	ChartRepo HelmChartRepo `yaml:"chart-repo"`
```

Comment:
```suggestion
	ChartRepo HelmChartRepo `yaml:"chartRepoURL"`
```

Reply:
as you have pointed under, `url` is a member of `HelmChartRepo` structure. The idea is to have the config expandable for future improvements, e.g. if we need to provide more infos about a given helm repository - authentication mechanism, some filtering options, etc.

Reply:
@pedjak maybe then just call it `chartRepo` instead of `chart-repo`?